### PR TITLE
Configuration unification

### PR DESCRIPTION
**Pull request by przemyslaw-jasinski**

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -10,7 +10,8 @@ project_settings:
   backup_project_id: '<your-project-id>'
   restoration_project_id: '<your-project-id>'
   authorized_requestor_service_accounts: ['sa-eligible-to-read-data@your-project-id.iam.gserviceaccount.com']
+  debug_mode: True
```
Comment (marcin-kolda) [inline config/config.yaml]: By default I would change debug_mode as False

Comment (marcin-kolda) [reply config/config.yaml]: In fact you should use prod configuration as the default one

**Comment by marcin-kolda** [comment]
What about documentation changes?

File: config/config.yaml (hunk 1 above)
Comment (przemyslaw-jasinski) [reply config/config.yaml]: Done, both :)

**Pull request merged**